### DISCUSSION TITLE
I wasn't able to download an opera from RTPplay

### DISCUSSION THREAD

**Discussion by papfneto** [aclap-dev/video-downloadhelper#536] [Help and Q&A]
### Your question

Hi guys!
Today I try to download an opera from RTPplay and I wasn't able to do it.
I don't understand why, since I've always managed to do it in the past.
It tells me: "Error: Native port disconnected". Can someone help me?!
Thank you so much!!!

### Addon & CoApp version and details

{
  "version": "9.0.1.13",
  "target": "google",
  "channel": "stable",
  "page_url": "undef",
  "error": {
    "details": "Error: Native port disconnected",
    "downloadable_id": "downloadable_4007765604137946",
    "error": "coapp_failure",
    "id": "coapp_failure_12e9631e-d0d7-406a-a892-7e258419b1ea"
  }
}

**Comment by paulrouget** [comment]
Does restarting your browser helps?

**Reply by papfneto** [reply]
Hi! No, it doesn't!

**Comment by GrampaWildWilly** [comment]
I love opera.  Please post the URL of the web page on which the opera is hosted.  No, don't tell me to pick any one of them.  Tell me the one you had trouble with.

**Reply by papfneto** [reply]
Hi! Here it goes:
https://www.rtp.pt/play/palco/p13415/viva-la-mamma

**Reply by GrampaWildWilly** [reply]
Thank you!

I was laughing at the synopsis of the opera.  I want to see this.  But every time I tried to launch playback via the small icon below the synopsis, it just sent me back to the same page.  The Network Monitor shows a file with the name .mp4, but it's not a video.  It's a json.  Maybe that's why VDH can't download anything.

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/d917f98f-2a77-4637-b4bc-5d3c2441ab1e)

It looks to me like the web site is malfunctioning.  Have you been able to play the opera in your browser?  There's a few operas listed across the bottom of the page.  I tried La Wally & got the same problem.  Maybe I'm being geoblocked & it's just not alerting me to that problem.  This web site is acting very weird for me.
